### PR TITLE
PSQLADM-426 Provide instructions to install Percona Scheduler from a …

### PR DESCRIPTION
**Pull request by patrickbirch**
…Percona respository

MM docs/install-v2.md
M  docs/uninstall-psql.md
M  docs/upgrade-psql.md

### CODE REVIEW

File: docs/uninstall-psql.md [hunk 1]
```diff
@@ -1,13 +1,119 @@
...
 
-```bash
-$> sudo apt-get remove proxysql2
+## Uninstalling using APT in Debian-based Linux distributions
+
+The Advanced Package Tool (APT) handles the software installation and removal on Debian and Debian-based Linux distributions.
+
+### Listing package contents using APT
+
+To find information before uninstalling or removing the `ProxySQL2` package, you can use the following command:
```
Comment (alina-derkach-oaza) [inline docs/uninstall-psql.md:11]: you can use the following command: -> use the following command

Comment (patrickbirch) [reply docs/uninstall-psql.md:11]: done

File: docs/uninstall-psql.md [hunk 2]
```diff
@@ -1,13 +1,119 @@
...
+The output would be the following:
+
+```text
+Sorting... Done
+Full Text Search... Done
+proxysql2/unknown,now 2.4.4-1.1.focal amd64 [installed]
+  High performance MySQL proxy
+```
+
+To find more information about `ProxySQL2`, including the installation, download size, sources for the packages, content description, and dependencies.
```
[the remark this answers is not in the document]
Comment (patrickbirch) [reply docs/uninstall-psql.md:26]: rewritten

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**